src/main/java/rx/schedulers/TestScheduler.java

```diff
@@ -57,9 +57,9 @@ public String toString() {
         @Override
         public int compare(TimedAction action1, TimedAction action2) {
             if (action1.time == action2.time) {
-                return Long.valueOf(action1.count).compareTo(Long.valueOf(action2.count));
+                return Long.valueOf(action1.count).compareTo(action2.count);
```

Comment:
To avoid misunderstanding, don't use `Long.compare()` because we need to be Java 6 compatible and that method is not available in it.